plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/WooCommercePayments.php

```diff
@@ -133,7 +133,16 @@ public static function is_installed() {
 	}
 
 	/**
-	 * Check if WooCommerce Payments is connected.
+	 * Check if the WooPayments plugin is active.
+	 *
+	 * @return bool
+	 */
+	public static function is_wcpay_active() {
```

Comment:
```suggestion
	public static function is_active() {
```

Reply:
or `is_plugin_active`

Reply:
The `Task` main class already has `is_active` to check if the task list item is active. I wanted to avoid any confusion about what we are checking.

Reply:
ok, good point! Closing this one